generate_tuxsuite.py

```diff
@@ -70,8 +73,18 @@ def emit_tuxsuite_yml(config, tree):
             if "make_variables" in build:
                 current_build.update(
                     {"make_variables": build["make_variables"]})
-            tuxsuite_buildset["sets"][0]["builds"] += [current_build]
 
+            if "defconfig" in str(build["config"]):
+                defconfigs += [current_build]
+            else:
+                allconfigs += [current_build]
```

Comment:
`.append` wouldn't create a new array

Reply:
Are you talking about doing this change?

```diff
diff --git a/generate_tuxsuite.py b/generate_tuxsuite.py
index ecccea4..e4e8ad0 100755
--- a/generate_tuxsuite.py
+++ b/generate_tuxsuite.py
@@ -75,9 +75,9 @@ def emit_tuxsuite_yml(config, tree):
                     {"make_variables": build["make_variables"]})

             if "defconfig" in str(build["config"]):
-                defconfigs += [current_build]
+                defconfigs.append(current_build)
             else:
-                allconfigs += [current_build]
+                allconfigs.append(current_build)

     tuxsuite_buildset["sets"][0]["builds"] = defconfigs
     if allconfigs:
```

Reply:
yes

Reply:
Done.